Source/WebCore/html/shadow/SpatialImageControls.cpp

```diff
@@ -115,10 +116,20 @@ void ensureSpatialControls(HTMLImageElement& imageElement)
         if (hasSpatialImageControls(*element))
             return;
 
+        StringBuilder controlStyleBuilder;
+        controlStyleBuilder.append("display: flex; flex-direction: column; justify-content: space-between; position: relative; box-sizing: border-box;"_s);
```

Comment:
We should use StyledElement::setInlineStyleProperty instead of building a string to parse.

Reply:
Good call!